### PR TITLE
Palindrome 28-09-24-1

### PR DESCRIPTION
**Pull request by enekesabel**
#### Test cases

- [x] "mom" returns true
- [x] "Mom" returns true
- [x] "MoM" returns true
- [x] "Momx" returns false
- [x] "xMomx" returns true
- [x] "Was It A Rat I Saw" returns true
- [x] "Never Odd or Even" returns true
- [x] "Never Odd or Even1" returns false 
- [x] "1Never Odd or Even1" returns true

### Additionally

- [x] test edge-cases
- [x] I have committed on every single transition from red to green to refactor
- [x] Once I have made the aforementioned tests pass, I have refactored my test specifications to use it.each() to perform parameterization 
- [x] There is no duplication in my test code or my production code

### CODE REVIEW

File: ThePhasesOfCraftship/2_best_practice_first/testingBasics/exercises/1_Red_Green_Refactor/1_2_Palindrome/src/index.ts [hunk 1]
```diff
@@ -0,0 +1,6 @@
+export const isPalindrome = (str: string): boolean =>{
+     // Remove spaces and convert to lowercase
+    const cleanedStr = str.replace(/\s+/g, '').toLowerCase();
```
Comment (jvhellemondt) [inline ThePhasesOfCraftship/2_best_practice_first/testingBasics/exercises/1_Red_Green_Refactor/1_2_Palindrome/src/index.ts]: What is a clean string? Will you still remember what `\s+` does in the future?

**Review by jvhellemondt — APPROVED** [review]
Small note on readability. Looks good!

File: ThePhasesOfCraftship/2_best_practice_first/testingBasics/exercises/1_Red_Green_Refactor/1_2_Palindrome/src/index.ts (hunk 1 above)
Comment (enekesabel) [reply ThePhasesOfCraftship/2_best_practice_first/testingBasics/exercises/1_Red_Green_Refactor/1_2_Palindrome/src/index.ts]: You're totally right, I'm already not 100% sure 😆 

I changed it in the last commit: https://github.com/enekesabel/the-software-essentialist/pull/2/commits/d901763302ab5921b6fdab550d04a3605ca7cc5c
